web/server/main.py

```diff
@@ -35,6 +40,25 @@ def shutdown_event() -> None:
     app.state.watch_task.cancel()
 
 
+@app.exception_handler(ApiException)
+async def handle_api_exception(_: Request, e: ApiException) -> JSONResponse:
+    return JSONResponse(
+        status_code=e.status_code,
+        content=e.to_dict(),
+    )
+
+
+@app.exception_handler(Exception)
+async def handle_uncaught_exeption(_: Request, e: Exception) -> JSONResponse:
+    return JSONResponse(
+        status_code=HTTP_422_UNPROCESSABLE_ENTITY,
```

Comment:
I'm not sure 422 is the right status code. I'd think 500 is still the correct code. Does the app do anything with these status codes?

Reply:
500 does not allow payload

Reply:
made it 500 for all uncaught